sentry-core/src/main/java/io/sentry/core/transport/HttpTransport.java

```diff
@@ -1,25 +1,16 @@
 package io.sentry.core.transport;
 
 import static io.sentry.core.ILogger.log;
-import static io.sentry.core.SentryLevel.DEBUG;
```

Comment:
I think we need to bring these imports back so spotless let's us thru

Reply:
spotless is good here, is CI failing?